src/Turing.jl

```diff
@@ -18,6 +18,10 @@ import StatsBase
 import Printf
 import Random
 
+using ADTypes: ADTypes
+
+default_adtype() = ADTypes.AutoForwardDiff(; chunksize=0)
```

Comment:
Does anyone know why we specify `chunksize=0` instead of just using `chunksize=nothing`?